app/GenerateGerberTemplates.hs

```diff
@@ -0,0 +1,167 @@
+module GenerateGerberTemplates (
+gerberHeader,
+gerberFooter,
+drawings
+) where
+
+gerberHeader :: String
+gerberHeader = "\
+  \G75*\n\
```

Comment:
You can also try adding Data.Gerber.Pretty module which would allow generating this with sort-of DSL (check out for example https://github.com/hackerspace/gcodehs/blob/master/examples/Generate.hs 

Pretty modules often contain pretty printer for related datatype so you can instead write `[ Toolchange 2, EndOfFile]` and then use pretty printer to generate code. On top of this you can base some shorthand function like `tool = Toolchange` and so on.